doc/writeYourOwn/intro.rst

```diff
@@ -0,0 +1,18 @@
+Writing Your Own Cube Browser
```

Comment:
We seem to have gained a new folder and file! And the file seems to be about identical to the file below...